packages/components/src/feature_alert/index.module.scss

```diff
@@ -0,0 +1,92 @@
+@import 'nebenan-ui-kit';
+
+$feature-alert-arrow-size: 16px;
```

Comment:
variable not needed imo

Reply:
We use it 2 times, and it explains the reason behind 16px somehow.